extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -261,56 +259,56 @@ function AchTest() {
 
 	global $wgUser, $wgExternalSharedDB;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)